apps/frontend/src/libs/components/icon/libs/maps/icon-name-to-svg.map.ts

```diff
@@ -26,6 +28,7 @@ const iconNameToSvg: Record<
 	IconName,
 	React.FC<React.SVGProps<SVGSVGElement>>
 > = {
+	arrow_next: ArrowNext,
```

Comment:
```suggestion
	arrowNext: ArrowNext,
```
Keep the same naming styles